public/course_lecture.php

```diff
@@ -102,3 +102,5 @@
     }
     ?>
 </div>
+
+<?php include "templates/footer.php"; ?>
```

Comment:
need this to clear error msg (we have several files that don't include the footer we might have to fix it)